src/fondant/cli.py

```diff
@@ -63,13 +66,14 @@ def entrypoint():
     )
     subparsers = parser.add_subparsers()
     register_explore(subparsers)
+    register_execute(subparsers)
     register_compile(subparsers)
     register_run(subparsers)
 
     sys.path.append(".")
 
     # display help if no arguments are provided
-    args = parser.parse_args(sys.argv[1:] or ["--help"])
+    args, _ = parser.parse_known_args(sys.argv[1:] or ["--help"])
```

Comment:
this is needed to avoid parsing the kubeflow arguments when calling `fondant execute main`

Reply:
This will ignore unknown arguments for other commands as well though, which might not be what we want. Maybe we can try something like:

```python
args, _ = parser.parse_known_args(sys.argv[1:] or ["--help"])
if args.func != execute:
    args = parser.parse_args(sys.argv[1:] or ["--help"])
```